content/docs/welcome-guide/requirements.md

```diff
@@ -6,30 +6,36 @@ weight: 300
-prerequisites and configuration](#software-prerequisites)
-section, each prerequisite is listed along with any specific configuration steps
-required.
+**Open 3D Engine (O3DE)** has a minimum set of hardware and software requirements for development, as outlined in the following sections. In the [Software prerequisites and configuration](#software-prerequisites) section, each prerequisite is listed along with any specific configuration steps required.
 
 ## Hardware requirements
 
-Minimum hardware requirements for development include the following:
-
-+ Quad-core Intel or AMD processor, 2.5 GHz or faster
-+ 8 GB RAM
-+ DirectX 12, Vulkan-compatible, or Metal-compatible video card with 2 GB VRAM minimum
-  + 4 GB VRAM recommended
-  + NVIDIA GeForce GTX 1060 or better, driver version 471.11 or later recommended
-  + AMD Radeon Pro 560 or better
-  + Intel HD 630 or better
-  + Shader Model 6.2 (or Shader Model 6.3 to use raytracing features)
-+ 40 - 100 GB of free disk space
-+ 1366 x 768 px screen resolution
+The following sections specify the minimum and recommended system hardware required to install or build **O3DE Editor** and related tools, and to develop O3DE projects.  RAM and free disk space requirements are dependent on the options that you select when configuring your project in O3DE.  Performance will vary based on the complexity of your project's levels.  
 
 {{< note >}}
-RAM and free disk space requirements are dependent on the options that you select when configuring your project in O3DE.
+Building the engine from source requires 2GB _per core_ in addition to your system's running requirements.  However, it is possible to limit the number of cores that are allocated to the build process.  
+
+Installing the engine with the [pre-built installer](https://o3debinaries.org/download/windows.html) requires less disk space and RAM than building from source.
 {{< /note >}}
 
+### Minimum hardware specification
+
+| Hardware Component | Minimum Specification |
+| - | - |
+| CPU | Quad-core (4 cores) 64-bit x86-capable Intel or AMD processor, 2.5 GHz, with the SSE 4.1 SIMD instruction set |
+| RAM | 16 GB RAM (8GB RAM may be acceptable if you limit the number of processes used for compilation) |
+| GPU | - DirectX 12 or Vulkan-compatible video card <br> - 2 GB VRAM <br> - Shader Model 6.2 (or Shader Model 6.3 to use raytracing features) <br> - NVIDIA GeForce GTX 1060, driver version 471.11 or later recommended _or_ <br> - AMD Radeon Pro 560 _or_ <br> - Intel HD 630 <br> |
+| Hard Disk | 40 GB (using the [pre-built installer](https://o3debinaries.org/download/windows.html)) _or_ 100+ GB (depending on project configuration) of free disk space |
+
+### Recommended hardware specification
+
+| Hardware Component | Recommended Specification |
+| - | - |
+| CPU | Hexa-core (6 cores) 64-bit x86-capable Intel or AMD processor, 2.5 GHz, supports the SSE 4.1 SIMD instruction set |
```

Comment:
Do we need to add "or better" or "or faster" here? While its implied, as written, this can be interpreted as you need a machine that matches this specification.

Reply:
Do you suggest we add that phrasing to each component of the recommended specs and leave it out of minimum specs?

Reply:
Maybe classify the two sets of specs as something  like "Minimum Required" and "Minimum Recommended".